tests/drivers/flash/common/src/main.c

```diff
@@ -47,21 +47,27 @@
 static const struct device *const flash_dev = TEST_AREA_DEVICE;
 static struct flash_pages_info page_info;
 static uint8_t __aligned(4) expected[EXPECTED_SIZE];
-static const struct flash_parameters *flash_params;
+static struct flash_parameters flash_params;
 static uint8_t erase_value;
 
 static void *flash_driver_setup(void)
 {
 	int rc;
+	uint8_t erase_value;
```

Comment:
Why driver user needs to know erase_value? Or what is the reason to introduce it?

Reply:
No. this is test specific so that we could check whether we have change in buffer we used for reading.